lib/aws-record/record/table_config.rb

```diff
@@ -59,7 +70,9 @@ module Record
     #       :title,
     #       hash_key:  :forum_uuid,
     #       range_key: :post_title,
-    #       projection_type: "ALL"
```

Comment:
I don't see any related code changes with this doc update (besides tests), so I'm assuming this is for updating docs to reflect current behavior?

Reply:
Correct, that's actually a doc bug.